### ISSUE TITLE
Warp app icon and Cursor app icon are too similar

### ISSUE THREAD

**Issue by twardoch** [getcursor/cursor#2391]
The app icons for the Warp terminal and the Cursor code editor are too similar. It’s easy to confuse them when you work quickly. Would it be possible that you guys sort it out? 

![scr- 2024-11-16 at 00 08 38](https://github.com/user-attachments/assets/248b29ac-fb22-4e2d-baf6-18246723ac4f)

**Comment by oraclesoon** [comment]
I think you can right-click on the icon, view Properties and change to the icon of your liking.

**Comment by twardoch** [comment]
That's right. However, one of the best terminal apps and one of the best code editors on macOS will probably be used together by many people.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
